lib/fruitstrap.js

```diff
@@ -0,0 +1,52 @@
+import { exec } from 'teen_process';
+import logger from './logger';
+
+const FRUITSAP_PATH = 'fruitstrap';
+
+class Fruitstrap {
+ 
+  constructor (udid, fruitstrapPath) {
+    this.udid = udid;
+    this.fruitstrapPath = fruitstrapPath || FRUITSAP_PATH;
```

Comment:
I want to throw an exception here is path is invalid. Do you know trick to call async `fs.hasAccess` function in constructor?